build_debian.sh

```diff
@@ -318,8 +320,15 @@ sudo LANG=C chroot $FILESYSTEM_ROOT useradd -G sudo,docker $USERNAME -c "$DEFAUL
 echo "$USERNAME:$PASSWORD" | sudo LANG=C chroot $FILESYSTEM_ROOT chpasswd
 
 ## Create redis group
-sudo LANG=C chroot $FILESYSTEM_ROOT groupadd -f redis
-sudo LANG=C chroot $FILESYSTEM_ROOT usermod -aG redis $USERNAME
+sudo LANG=C chroot $FILESYSTEM_ROOT groupadd -f -g $REDIS_USER_GID $REDIS_GROUP
+sudo LANG=C chroot $FILESYSTEM_ROOT usermod -aG $REDIS_GROUP $USERNAME
+
+# Ensure redis gid is 1001.
```

Comment:
>1001 [](http://example.com/codeflow?start=22&length=4)

Do not use magic number in comment, it is already a named constant.

Reply:
removed